src/textual/color.py

```diff
@@ -447,10 +449,10 @@ def parse(cls, color_text: str | Color) -> Color:
         ) = color_match.groups()
 
         if rgb_hex_triple is not None:
-            r, g, b = rgb_hex_triple
+            r, g, b = rgb_hex_triple  # type: ignore[misc]
```

Comment:
Is this because MyPy has no way of knowing that the regex match group can be unpacked into this?

Reply:
It's because Mypy doesn't approve of unpacking strings. Somebody decided that was likely to be an error. Not that I've ever done it accidentally.